docs/example.md

```diff
@@ -146,46 +227,23 @@ This example is for a connector to Amazon Athena.
 })
 ```
 
-## ![6]({{ site.baseurl }}/assets/pce-6.png) Connection Matcher
-
-This component defines how connections are matched, and the default settings are already performance optimized. It is rare that you will need to include a custom <span style= "font-family: courier new">connection-matcher</span> section in your TDR file.  
-
-## ![7]({{ site.baseurl }}/assets/pce-7.png) Connection Normalizer
-
-The component defines what makes up a unique connection. This is best implemented directly in the XML file. Writing the required attributes list in XML is more performant than using a JavaScript file, and is recommended for most connectors.
-
-```
-<required-attributes>
-        <setImpersonateAttributes/>
-        <attribute-list>
-          <attr>server</attr>
-          <attr>port</attr>
-          <attr>dbname</attr>
-          <attr>username</attr>
-          <attr>password</attr>
-          <attr>one-time-sql</attr>
-        </attribute-list>
-</required-attributes>
-```
-`<setImpersonateAttributes/>` and `<attr>one-time-sql</attr>` add support for impersonate attributes and initial sql respectively, and should be in every connector.
-
-## ![8]({{ site.baseurl }}/assets/pce-8.png) Connection - ODBC String or JDBC URL 
+## ![6]({{ site.baseurl }}/assets/pce-8.png) Connection example
 
 The Tableau Connection Resolver file (.tdr) generates an ODBC ConnectString or a JDBC Connection URL, which you can find in tabprotosrv.txt.
 
 For ODBC, search for <span style= "font-family: courier new">ConnectString</span> to find something like this example:
 
 ```
-ConnectString: DRIVER={PostgreSQL Unicode(x64)};SERVER=postgres;PORT=5432;DATABASE=TestV1;UID=test;PWD=********;BOOLSASCHAR=0;LFCONVERSION=0;UseDeclareFetch=1;Fetch=2525
+ConnectString: DRIVER={PostgreSQL Unicode(x64)};SERVER=postgres;PORT=5432;DATABASE=TestV1;UID=test;PWD=********;BOOLSASCHAR=0;LFCONVERSION=0;UseDeclareFetch=1;Fetch=2048
 ```
 
 For JDBC, search for <span style= "font-family: courier new">Connection URL</span> to find something like this example:
 
 ```
-JDBCProtocol Connection URL: jdbc:postgresql://postgres:5342/TestV1?user=test&password=********
+JDBCProtocol Connection URL: jdbc:postgresql://postgres:5342/TestV1
 ```
 
-## ![9]({{ site.baseurl }}/assets/pce-9.png) Tableau Dialect File
+## ![7]({{ site.baseurl }}/assets/pce-9.png) 
```

Comment:
I think this should still say 'Tableau Dialect File', likely just a merge issue.

Reply:
Oops , looks like in in the Pull request for the gh-pages 2020.4 branch the 'Tableau Dialect File' was removed mistakenly . Fixing it.